### DISCUSSION TITLE
How to set graph.name during keras export?

### DISCUSSION THREAD

**Discussion by dbolotov** [onnx/tensorflow-onnx#1828] [General]
Hi,

Is there a way to rename the graph while converting and saving a keras model with `tf2onnx.convert.from_keras()`?

As shown in the script below, `model_proto.graph.name` is "tf2onnx" when using `tf2onnx.convert.from_keras()`. To rename and save, I currently set the name using `model_proto.graph.name = 'new_model_name'` after converting, and then save with `onnx.save`. 

Is it possible to set the name directly with `tf2onnx.convert.from_keras()`? 

```python
import tensorflow as tf
import tf2onnx
from keras.models import load_model

model = load_model("keras_model.h5")
spec = (tf.TensorSpec((None, 97, 80, 3), tf.float32, name="input"),)

# Save using `output_path` in from_keras():
model_proto, _ = tf2onnx.convert.from_keras(model = model, input_signature = spec,
                                            output_path = "model01.onnx")

print(model_proto.graph.name) # returns `tf2onnx`


# Save with onnx after setting graph.name
model_proto, _ = tf2onnx.convert.from_keras(model = model, input_signature = spec)

import onnx
model_proto.graph.name = 'new_model_name'
print(model_proto.graph.name) # returns `new_model_name`
onnx.save(model_proto, "model02.onnx")
```

Thank you.

**Comment by ThomasManz** [comment]
Hi,

I would suggest the following change to take the Keras model name as ONNX graph name:

old:
`model_proto = onnx_graph.make_model("converted from {}".format(name),`

new:
`model_proto = onnx_graph.make_model("converted from Keras/Tensorflow", graph_name=name,`

https://github.com/onnx/tensorflow-onnx/blob/82ae7c4659ab9ece121a6414ee037d68b6b2d907/tf2onnx/convert.py#L171
